### ISSUE TITLE
Hold Codewind planning in the open

### ISSUE THREAD

**Issue by johnmcollier** [eclipse/codewind#1110]
As Codewind continues to grow and evolve as an open source project, we should be holding its backlog & feature planning in the open. We currently use `dev-ex-planning` in the IBM GHE for Codewind planning, and I think we should be able to migrate _most_ of it to an open repo. This planning repo / zenhub board should be linked to / reachable from the Codewind repo, so that users can easily find it.

**Questions:**
1) Any objections to migrating the IBM GHE `dev-ex-planning` and corresponding Zenhub board we use to a public Github?
2) Do we need to keep using something like Jira or Aha! for planning?
    - After speaking with @rtaniwa and @deboer-tim on this, I think the answer is **no**
3) What should we name the planning repo and where to put it?
     - Thinking `codewind-planning`.
     - Could put it under `github.com/eclipse` or `github.com/codewind-resources`. The latter has the advantage that non-committers like @malincoln or @rtaniwa can easily interact with issues on the Zenhub board
4) Anything else?

**Comment by rtaniwa** [comment]
I am on board. I do want the ability to manage issues though, so prefer somewhere where I don't require being a committer. Suggested name and location are both fine for me. 

Do we need any clean-up before the move so we aren't moving stuff out there that is unneeded? 
Do we expect the community to open up issues directly for planning consideration, or will those flow naturally through a git issue in the main codewind repo?

**Comment by malincoln** [comment]
No objections here.  Codewind-planning sounds good to me.  I like the idea of the codewind-resources for easier interaction for non-committers too.  Looks like you've covered all areas as I can't think of anything else.  Thanks!

**Comment by johnmcollier** [comment]
@rtaniwa Yeah, probably best to do a look through of `dev-ex-planning` before moving to see if any clean up is needed. In terms of community participation I generally wouldn't expect the community to open up epics in `codewind-planning`, **but** they should be able to if they feel the need to and we can then prioritize as needed (or close/move them out if it's not relevant).

@malincoln Great! I'll get a repo created in `codewind-resources` and we can look at moving over after the release next week.

**Comment by rtaniwa** [comment]
Also during a discussion on the backlog call, @deboer-tim raised whether or not we could actually do a move of existing issues, and if so would they retain history etc. Not sure if we know of that. I will work with Tim and others to do a clean-up. Is there a timeframe for when you would like to target a move?

**Comment by johnmcollier** [comment]
Hmm, good point. I'm not actually aware of a utility that can easily move issues from GHE to GH, so it would have to be all manual work for the ones we want to move over (and history wouldn't be preserved). I suppose we could repurpose @jgwest's Github bot to do the move of the issues we want to keep, but he'd have to comment

I'm flexible on the timeframe, post 0.6.0 release seems like a good time (so week of the 25th perhaps?), but I'm fine with it being later if you all need more time.

**Comment by johnmcollier** [comment]
Don't think there are any outstanding questions that would need to be asked/answered on a tech-topics call, so removing the label
